src/backend/txn.go

```diff
@@ -564,6 +576,12 @@ func (txn *Txn) executeXA(req *xcontext.RequestContext, state txnXAState) error
 				}
 				break
 			}
+
+			if state == txnXAStateCommit && x != nil && txn.mgr.xaCheck != nil {
+				if err := txn.WriteXaCommitErrLog(back); err != nil {
```

Comment:
WriteXaCommitErrLog should place in Commit/Rollback when error happens.
But we must handle the case as follows(without Begin phase):
```
txn, err := txnMgr.CreateTxn(backends) 
txn.Execute(rctx)
txn.Commit() -- here the transaction missing txn.Begin(), so we will got an error when Commit, but we must filter this case to do WriteXaCommitErrLog.
```